src/main/target/FISHDRONEF4/target.h

```diff
@@ -20,9 +20,14 @@
 
 #pragma once
 
+#if defined(STACKX)
+#define TARGET_BOARD_IDENTIFIER "SXF4"
+#define USBD_PRODUCT_STRING     "Stack-X F4"
+#define SETVTX_ULTRALOWPOWER_DISARMED
```

Comment:
In order to move to fully configurable targets the addition of new target specific defines should be avoided, since they will have to be redone. A better way to do this is to add a configuration setting for it, and then use a target specific `config.c` to enable it.